ads/_config.js

```diff
@@ -172,6 +172,7 @@ export const adConfig = {
     masterFrameAccessibleType: 'google_network',
     fullWidthHeightRatio: 1.2,
     consentHandlingOverride: true,
+    renderStartImplemented: true,
```

Comment:
If the `xoriginIframeHandler` is created by the `AmpA4A` class. I don't think `this.baseInstance_.config` here is defined https://github.com/ampproject/amphtml/blob/724ce3222978ac55aeb6fe393150ee26e8f7e464/extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js#L221-L224

Could you let me know why `adsense` still has this configuration here when it is using the fastFetch. 

Any how I think we need to add `opt_letCreativeTriggerRenderStart` to the check above to fix the issue.

Reply:
resolved offline